erna/__init__.py

```diff
@@ -18,8 +18,32 @@ def build_path(row, path_to_data, extension):
     year = night[0:4]
     month = night[4:6]
     day = night[6:8]
-    return os.path.join(path_to_data, year, month, day, row.filename + extension)
-
+    res = os.path.join(path_to_data, year, month, day, row.filename + extension)
+    return res
+
+def test_drs_path(df, key):
+    numRows = len(df)
+    mask = np.ones(numRows);
+    for i in range(numRows):
+        drspath = fd[key][i]
+        if not os.path.exists(drspath):
+            mask[i] = 0
+    return df.groupby(mask)
```

Comment:
This function is very unefficient and unpythonic. The whole thing could be written as

```python
df['file_exists'] = df[key].apply(os.path.exists)
return df.groupby('file_exist')
```
But why would this function return a grouped object?